governance.Jenkinsfile

```diff
@@ -31,12 +43,28 @@ podTemplate(label: label) {
             currentBuild.result = "FAILURE"
             def body = "${currentBuild.currentResult} ${env.JOB_NAME}${env.BUILD_DISPLAY_NAME}: on branch: ${params.GIT_BRANCH}. See details: ${env.BUILD_URL}"
             emailext body: body, recipientProviders: [[$class: 'DevelopersRecipientProvider'], [$class: 'CulpritsRecipientProvider'], [$class: 'RequesterRecipientProvider']], subject: "${currentBuild.currentResult}: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]'"
+            
+            if(isMaster) {
+                sendSlackNotification(":hankey-fire:Governance validation failed on `${repositoryName}` repository!\nSee details: ${env.BUILD_URL}console")
+            }
         }
     }
 }
 
-def validateLinks(command) {
+def validateLinks(args, repositoryName) {
     workDir = pwd()
-    whiteList = "github.com,localhost,core-publish.kyma-system,kyma-integration,svc.cluster.local,calculate-promotion,kyma.local"
-    sh "docker run --rm -v $workDir:/mnt:ro dkhamsing/awesome_bot --allow-dupe --allow-redirect --skip-save-results --allow-ssl --white-list $whiteList `$command`"
+    sh "docker run --rm -v ${workDir}:/${repositoryName}:ro magicmatatjahu/milv:0.0.3 --base-path=/${repositoryName} ${args}"
```

Comment:
These arguments are not clear to me. For example, when you set it to `--ignore-internal`, how does the other application know the internal links to ignore?

Reply:
Please see: https://github.com/magicmatatjahu/milv